src/app/students/browsecredits/BackButton.tsx

```diff
@@ -0,0 +1,11 @@
+const Button = () => {
+	return (
+		<button
+			className={`h-[2.4375rem] w-[5.125rem] rounded-full bg-[#805DBE] text-white text-xs font-bold hover:bg-violet-700`}
```

Comment:
always using padding for buttons.  could you update this?

Reply:
updated to padding, included Link component.